mos_tests/murano/murano_test.py

```diff
@@ -0,0 +1,134 @@
+#    Copyright 2015 Mirantis, Inc.
+#
+#    Licensed under the Apache License, Version 2.0 (the "License"); you may
+#    not use this file except in compliance with the License. You may obtain
+#    a copy of the License at
+#
+#         http://www.apache.org/licenses/LICENSE-2.0
+#
+#    Unless required by applicable law or agreed to in writing, software
+#    distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+#    WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+#    License for the specific language governing permissions and limitations
+#    under the License.
+# from unittest import TestCase
+
+import pytest
+from random import randint
+from mos_tests.functions import os_cli
+
+pytestmark = pytest.mark.undestructive
+
+
+def rand_num(a=1, b=1000):
+    return randint(a, b)
+
+
+def user_env(user, password, project):
+    """Export this env if you need to perform actions from another user"""
+    env = 'export OS_TENANT_NAME="{2}" && ' \
+          'export OS_PROJECT_NAME="{2}" && ' \
+          'export OS_USERNAME="{0}" && ' \
+          'export OS_PASSWORD="{1}"'.format(
+            user, password, project)
+    return env
+
+
+def execute_on_all_controllers(env, cmd):
+    controllers = env.get_nodes_by_role('controller')
+    all_output = []
+    for controller in controllers:
+        with controller.ssh() as remote:
+            output = os_cli.CLICLient(remote).execute(cmd)
+            all_output.append(output)
+    return all_output
+
+
+@pytest.mark.xfail(run=False, reason="1551575")
```

Comment:
Xfail purpose is for checking correct behavior (for example, int('') should raise ValueError).
Here exception is not correct behavior of product under testing.